sap_hana/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,50 @@
+name: SAP HANA
+files:
+- name: sap_hana.yaml
+  options:
+  - template: init_config
+    options:
+      - template: init_config/db
+      - template: init_config/default
+  - template: instances
+    options:
+      - name: server
+        description: The hostname used to connect to the system.
+        required: true
+        value:
+          type: string
+      - name: port
+        description: The port used to connect to the system.
+        value:
+          type: integer
+          default: 30015
+      - name: username
+        description: The database user to authenticate as.
+        required: true
+        value:
+          type: string
+      - name: password
+        description: The password of `username`.
+        required: true
+        value:
+          type: string
+      - name: use_hana_hostnames
+        description: |
+          Whether or not to use the hostnames contained in HANA's monitoring
+          views instead of the Agent hostname.
+        value:
+          type: boolean
+          default: false
+          example: false
```

Comment:
Probably don't need example if you already have default?

Reply:
We actually need example, but default is not required indeed